Classes/Email/Salutation.php

```diff
@@ -109,15 +95,20 @@ public function createIntroduction(string $introductionBegin, \Tx_Seminars_OldMo
         if (!$event->hasDate()) {
             return $result;
         }
-
-        $result .= ' ' . sprintf($this->translator->translate('email_eventDate'), $event->getDate('-'));
+        $result .= ' ' . sprintf(
+            LocalizationUtility::translate(
```

Comment:
Let let's put this `translate` call into a single line to make this method less long (vertically).